task_10/src/History/History.js

```diff
@@ -0,0 +1,49 @@
+require('./History.scss')
+
+import Favourites from '../Favourites'
+
+export default class History extends Favourites {
```

Comment:
FYI: In general, for everyday use JS and classes it is ok. But remember, that in React, as example, you usually don't extending another components.

As I said, for now, it's ok.